packages/components/src/alignment-matrix-control/utils.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const GRID = [
+	[ 'top left', 'top center', 'top right' ],
+	[ 'center left', 'center center', 'center right' ],
+	[ 'bottom left', 'bottom center', 'bottom right' ],
+];
+
+// Stored as map as i18n __() only accepts strings (not variables)
+export const ALIGNMENT_LABEL = {
+	'top left': __( 'Top Left' ),
+	'top center': __( 'Top Center' ),
+	'top right': __( 'Top Right' ),
+	'center left': __( 'Center Left' ),
+	'center center': __( 'Center Center' ),
```

Comment:
nit: but I think we should `middle` for the vertical axis and `center` for the horizontal to avoid this "center center" repetition.

Reply:
As far as I'm aware, there's no difference in meaning between "middle" and "center", or at least not relating to axis. We could just have the displayed title be simply "Center".